doc/api/v8.markdown

```diff
@@ -21,6 +21,55 @@ Returns an object with the following properties
 }
 ```
 
+## getHeapSpaceStatistics()
+
+Returns statistics about the V8 heap spaces, i.e. the segments which make up
+the V8 heap. Order of heap spaces nor availability of a heap space can be
+guaranteed as the statistics are provided via the V8 `GetHeapSpaceStatistics`
+function.
+
+Possible result:
```

Comment:
Can I suggest using e.g. `Example result` here?  `Possible` suggests that it could be like this but also something completely different.